src/content/en/updates/2016/10/devtools-digest.md

```diff
@@ -0,0 +1,79 @@
+
+# DevTools Digest, October 2016 {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Hey hey, [Kayce](https://twitter.com/kaycebasques) here, back again for
+another digest to update you on what's new in DevTools since last month.
+
+[Crbug]: https://bugs.chromium.org/p/chromium/issues/list?can=1&q=component%3APlatform%3EDevTools+-status%3AUnconfirmed+-status%3AUntriaged++-status%3AAvailable+-status%3AAssigned+-status%3AStarted+-status%3AExternalDependency+-status%3ADuplicate+-status%3AWontFix+-status%3AArchived+&sort=-modified&colspec=ID+Pri+M+Stars+ReleaseBlock+Component+Status+Owner+Summary+OS+Modified&x=m&y=releaseblock&cells=ids
+
+## New features in the Console {: #console }
+
+In Chrome 56, which is currently Canary, the DevTools Console is powered by
+[CodeMirror](https://codemirror.net/). This enables a whole bunch of new
+features, like:
+
+**Syntax highlighting as you type**. Previously, DevTools could only
+highlight syntax after a code block had been evaluated.
+
+**Matching bracket highlighting**. Because no one got time for those annoying
+`Uncaught SyntaxError: Unexpected token {` errors.
+
+**Smart return**. When you type out a multi-line code block, DevTools
+now knows whether to create a new line or execute your code each time you
+press <kbd>Enter</kbd>. You can still force DevTools to create a new line by
+holding <kbd>Shift</kbd> before you press <kbd>Enter</kbd>.
+
+**Multiple cursors**. Hold <kbd>Command</kbd> (Mac) or
+<kbd>Control</kbd>+<kbd>Alt</kbd> (Windows, Linux) and then click.
+
+![multiple cursors](/web/updates/images/2016/10/multiple-cursors.gif)
+
+## Canary now highlights non-top contexts red
+
+If you've worked in the Console lately, you might have been bitten by a
+nasty little bug that was setting the [execution context
+selector](/web/tools/chrome-devtools/console/#execution-context) to values
+other than `top`.
+
+That bug should be fixed in Chrome 54, which is due to hit Stable any
+day now. And, just to be safe, the DevTools in Canary (Chrome 56) now warns
+you that you're not in the `top` context by highlighting the selector red.
+
+![non-top context highlighted
+red](/web/updates/images/2016/10/non-top-context.png)
+
+## New user agent: UC Browser
+
+You can now select UC Browser for iOS, Android, or Windows Phone from
+the **Network conditions** [drawer
+tab](/web/tools/chrome-devtools/settings#drawer-tabs).
+
+![UC Browser user agent](/web/updates/images/2016/10/uc-browser.png)
+
+## Sharing is caring
+
+As always, we'd love to hear your feedback or ideas on anything DevTools
+related.
+
+* Ping us at [ChromeDevTools](https://twitter.com/chromedevtools) on Twitter
+  for brief questions or feedback, or to share new ideas.
+* For longer discussions, the [mailing list](https://groups.google.com/forum/#!forum/google-chrome-developer-tools/topics) or [Stack Overflow](http://stackoverflow.com/questions/tagged/google-chrome-devtools) are your best bets.
```

Comment:
Nit: If both http and https domains are supported by an outbound link, preference for `//stackoverflow.com/....`